Tests/ResilientDecodingTests/ResilientDictionaryTests.swift

```diff
@@ -124,36 +124,4 @@ final class DictionaryTests: XCTestCase {
     #endif
   }
 
-  /**
```

Comment:
Why can this all be removed? What did we do to fix it?

Reply:
I figured out a way to not depend on pulling out `Decoder`s